numcosmo/math/ncm_rng.h

```diff
@@ -110,6 +111,13 @@ G_END_DECLS
 
 G_BEGIN_DECLS
 
+NCM_INLINE gdouble
+ncm_rng_chisq_gen(NcmRNG *rng, const gdouble nu)
+{
+  return gsl_ran_chisq(rng->r, nu);
+}
+
+
```

Comment:
Please add the corresponding documentation to the .c file (use the other function as an example).